src/logic/wallets/utils/network.ts

```diff
@@ -89,6 +89,6 @@ export const switchWalletChain = async (): Promise<void> => {
   } catch (e) {
     e.log()
     // Fallback to the onboard popup if switching isn't supported
-    await onboard().walletCheck()
+    await onboard().walletSelect()
```

Comment:
Is this product decision? Just asking because I couldn't find the requirement in the description

Reply:
Onboard was crashing the app unless `walletSelect()` is called first. I overlooked the removal of `walletCheck()`. I'll look into this again

Reply:
I've altered this to match the original functionality, following the [correct way of calling `walletCheck()`](https://docs.blocknative.com/onboard#onboard-user).